sympy/parsing/tests/test_sympy_parser.py

```diff
@@ -193,6 +190,18 @@ def test_recursive_evaluate_false_10560():
         assert parse_expr(text, evaluate=False) == parse_expr(result, evaluate=False)
 
 
+def test_issue__evaluateFalse_24288():
+    inputs = {'1<2' : Lt(1,2),
+              '1<=2' : Le(1,2),
+              '1>2' : Gt(1,2),
```

Comment:
Use a space after comma.

These tests need to have e.g. `Lt(1, 2, evaluate=False)` because otherwise they will just evaluate to True/False.